doozerlib/cli/release_gen_payload.py

```diff
@@ -1210,14 +1225,14 @@ def _find_rhcos_payload_entries(assembly_inspector: AssemblyInspector, arch: str
         return members, issues
 
     @staticmethod
-    def build_payload_istag(payload_tag_name: str, payload_entry: PayloadEntry) -> Dict:
+    def build_payload_istag(runtime, payload_tag_name: str, payload_entry: PayloadEntry) -> Dict:
```

Comment:
`runtime` is not used.

```suggestion
    def build_payload_istag(payload_tag_name: str, payload_entry: PayloadEntry) -> Dict:
```

Reply:
Updated